test/vast_parser.js

```diff
@@ -258,16 +258,16 @@ describe('VASTParser', function() {
 
       it('validate third extension', () => {
         ad1.extensions[2].attributes['type'].should.eql('Count');
-        ad1.extensions[2].children.should.have.length(1);
-        ad1.extensions[2].children[0].name.should.eql('#cdata-section');
-        ad1.extensions[2].children[0].value.should.eql('4');
+        ad1.extensions[2].children.should.have.length(0);
```

Comment:
I would prefer to move all tests related to `parseExtensions()` to a new test file called `ad_parser.js`. If you want you can rebase your changes on https://github.com/dailymotion/vast-client-js/pull/279 and implement this new file as jest (file should have `*.spec.js` name ending and be in `spec` folder with the other jest tests).